playbackScreen/src/main/kotlin/voice/playbackScreen/SelectChapterDialog.kt

```diff
@@ -1,53 +1,65 @@
 package voice.playbackScreen
 
 import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.lazy.LazyColumn
 import androidx.compose.foundation.lazy.itemsIndexed
 import androidx.compose.foundation.lazy.rememberLazyListState
-import androidx.compose.material.icons.Icons
-import androidx.compose.material.icons.outlined.Equalizer
-import androidx.compose.material3.AlertDialog
-import androidx.compose.material3.Icon
+import androidx.compose.foundation.shape.RoundedCornerShape
 import androidx.compose.material3.ListItem
 import androidx.compose.material3.ListItemDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.ModalBottomSheet
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.selected
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.unit.dp
+import voice.common.formatTime
 import voice.strings.R as StringsR
 
 @Composable
 internal fun SelectChapterDialog(
   dialogState: BookPlayDialogViewState.SelectChapterDialog,
   viewModel: BookPlayViewModel,
 ) {
-  AlertDialog(
+  ModalBottomSheet(
     onDismissRequest = { viewModel.dismissDialog() },
-    confirmButton = {},
-    text = {
+    content = {
       LazyColumn(
-        state = rememberLazyListState(initialFirstVisibleItemIndex = dialogState.selectedIndex ?: 0),
+        state = rememberLazyListState(initialFirstVisibleItemIndex = dialogState.selectedIndex?.minus(1)?.coerceAtLeast(0) ?: 0),
```

Comment:
Thanks! There is one bug: If the current chapter is not in the first 4-5 items, but for example on position 10, when opening the chapter list, the selected chapter is not visible.

Reply:
Hmm, I can't reproduce that, opened an audiobook, jumped 10 chapters forward, but the current chapter is still shown as the second visible item.

Reply:
Can you screenshot?

Reply:
After testing a bit, I think what's happening is that the LazyColumn always uses the maximum height of the ModalBottomSheet (i.e. screen height when it's fully expanded) and it's just moved into view when scrolling up.
As I rarely deal with Jetpack Compose, I'm a bit out of my depth here, maybe we can constrain the height of the column until the sheet is expanded? Or just fully expand it if the last chapter is selected?

Reply:
I tested it futher with more chapters and the UX of the bottom sheet is really not made for more than a handful of items.

An alternative to really add more estate would be to move it to its own screen.
[ux.webm](https://github.com/user-attachments/assets/862574e8-d664-4f98-a2c7-0fd83317ab1a)

Reply:
This seems to be a bug in the compose implementation of bottomsheets (I tested it with a 'normal' bottomsheet and a recyclerview, it works flawlessly). 

I'm personally not a huge fan of adding a new screen just for the chapters, since that causes the chapters to feel somewhat disconnected from the player. Also in that case we could also just always expand the sheet :)

Reply:
Solved by setting skipPartiallyExpanded = true.